lib/Models/ImageryLayerCatalogItem.js

```diff
@@ -1183,17 +1183,21 @@ ImageryLayerCatalogItem.enableLayer = function(
             } else if (!defined(e.statusCode) && defined(e.target)) {
```

Comment:
This condition looks like it's the wrong way around. Is this meant to be `defined(e.statusCode) && !defined(e.target)`? Just from the comment it looks more reasonable to be that.

Reply:
Looks alright to me.

>   // This is a failed image element, which means we got a 200 response but
>    // could not load it as an image.

What happens is the result of the xhr is set to an img element as a blob. But the img element errors out because the blob is invalid. The `e.target` is the img element that errored.

https://github.com/CesiumGS/cesium/blob/1.67/Source/Core/Resource.js#L1800

Reply:
Ok. Thanks for the explanation.